Rakefile

```diff
@@ -29,7 +28,9 @@ namespace :resque do
     }
   end
 
-  task :workers => :clean_working_job_ids # add job cleaning as a pre-requisite
+  # Add job cleaning as a pre-requisite
+  task work: :clean_working_job_ids
+  task workers: :clean_working_job_ids
```

Comment:
If we do `clean_working_job_ids` for `:work` do we still need it for `workers`? 

Because `workers` calls `work` internally as far as I know and that should trigger the `clean_working_job_ids` for all workers automatically. I haven't tested this by the way, my memory is hazy, it has been sometime since I worked on this rake task.

Reply:
`workers` seems to have as prerequisite `work` so I am going to remove it as requested at your risk